### PR TITLE
Ps post can be liked 144 updated

### PR DESCRIPTION
**Pull request by Tibor22**
PostLike functionality has been added alongside with delete on cascade

### CODE REVIEW

File: prisma/migrations/20220708105026_submission_table_added_after_merge/migration.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+*/
+-- DropForeignKey
+ALTER TABLE "Submission" DROP CONSTRAINT "Submission_cohortExerciseId_fkey";
+
+-- DropForeignKey
+ALTER TABLE "Submission" DROP CONSTRAINT "Submission_userId_fkey";
+
+-- DropTable
+DROP TABLE "CohortExercise";
```
Comment (dearshrewdwit) [inline prisma/migrations/20220708105026_submission_table_added_after_merge/migration.sql:15]: What is the reasoning for this?

File: src/controllers/post.js [hunk 2]
```diff
@@ -85,3 +85,24 @@ export const deletePost = async (req, res) => {
     return sendDataResponse(res, 400, { err: err.message })
   }
 }
+
+export const updateLike = async (req, res) => {
+  const postId = Number(req.params.id)
+  let { active, postLikeId, userId } = req.body
```
Comment (dearshrewdwit) [inline src/controllers/post.js:91]: Prefer `const` not `let`, and transform the values when needed - 

```js
    const updateLike = await PostLike.fromJson(
      active,
      Number(userId),
      postId,
      Number(postLikeId)
    )
```

**Review by dearshrewdwit — DISMISSED** [review]
Looks great - main question is about the destructive drop table.

File: prisma/migrations/20220708105026_submission_table_added_after_merge/migration.sql (hunk 1 above)
Comment (Tibor22) [reply prisma/migrations/20220708105026_submission_table_added_after_merge/migration.sql:15]: what should i do ?

Comment (dearshrewdwit) [reply prisma/migrations/20220708105026_submission_table_added_after_merge/migration.sql:15]: Sure. The point here is to double check schema changes, especially if they are lossy.

In this case, this change shouldn't be here - unless you have a reason to include it.

**Review by dogezen — APPROVED** [review]

**Review by dogezen — APPROVED** [review]

**Pull request merged**